_security/authentication-backends/openid-connect.md

```diff
@@ -334,6 +334,22 @@ opensearch.requestHeadersAllowlist: ["Authorization", "security_tenant"]
 To include OpenID Connect with other authentication types in the Dashboards sign-in window, see [Configuring sign-in options]({{site.url}}{{site.baseurl}}/security/configuration/multi-auth/).
 {: .note } 
 
+
+#### Session management with additional cookies
+
+To improve session management—especially for users who have multiple roles assigned to them—Dashboards provides an option to split cookie payloads into multiple cookies and then restitch the payloads when receiving them. This can help prevent larger OpenID Connect assertions from exceeding size limits for each cookie. The two settings in the following example allow you to set a prefix name for additional cookies and specify the number of them. The default number of additional cookies is three:
+
+```yml
+opensearch_security.openid.extra_storage.cookie_prefix: security_authentication_oidc
+opensearch_security.openid.extra_storage.additional_cookies: 3
+```
+
+Note that reducing the number of additional cookies can cause some of the cookies in use before the change to stop working. We recommend establishing an appropriate number of additional cookies and leaving the setting in that configuration.
```

Comment:
"appropriate" might be subjective. Maybe "fixed"? "not changing" instead of "leaving"?